### ISSUE TITLE
Ads shown are not being correctly counted, no BAT received

### ISSUE THREAD

**Issue by Sh1d0w** [brave/brave-browser#8202]
## Description 
Since version 1.3.115 which was released yesterday I do not gain BAT from ads anymore. I can see the ads notifications appearing, I am even clicking them, but the overall count of "Ad notifications received this month" does not change (the ads are registered correctly in 7-day ads history) and no BAT has been gained.

<img width="620" alt="Screenshot 2020-02-12 at 6 10 04" src="https://user-images.githubusercontent.com/5074917/74302233-68477700-4d5e-11ea-8e86-e13b3a3b6525.png">

<img width="919" alt="Screenshot 2020-02-12 at 6 10 11" src="https://user-images.githubusercontent.com/5074917/74302238-6d0c2b00-4d5e-11ea-938a-607340d275ce.png">

## Steps to Reproduce

   1.  Update to Brave 1.3.115
   2.  See some ads
   3. Check Brave Rewards Tab

## Actual result:
The total ads count does not increase, no BAT received.

## Expected result:
The total ads count should increase and should receive some BAT.

## Reproduces how often: 
Every time.

## Brave version (brave://version info)

Brave | 1.3.115 Chromium: 80.0.3987.87 (Official Build) (64-bit)
-- | --
Revision | 449cb163497b70dbf98d389f54e38e85d4c59b43-refs/branch-heads/3987@{#801}
OS | macOS Version 10.15.3 (Build 19D76)

## Version/Channel Information:
It happens on stable channel

**Comment by Sh1d0w** [comment]
Seems like a lot of users are having the same problem:

https://www.reddit.com/r/brave_browser/comments/f2lfk2/bat_not_increasing/
https://www.reddit.com/r/brave_browser/comments/f2is23/brave_not_counting_ads_seen_properly/

**Comment by jonathansampson** [comment]
I suspect this is due to some network hiccups during ad confirmation. No need to be alarmed though, as Brave will attempt to confirm those ad views again over the next 24 hours or so. Please check back in a couple of days to let us know if the issue has resolved itself.

**Comment by Sh1d0w** [comment]
@jonathansampson Hey, unfortunately the issue does not seem to resolve. It's been more than 24 hours since I got some ads that are not reflected in the counter, not I get BAT for them.

According to my Ads history since 11th of Feb I've received 8 ads, however the counter is stuck on 2 since then.

Let me know if I can help with something in order to better understand the issue.

**Comment by jonathansampson** [comment]
Thank you for the update, @Sh1d0w. I'll check-in with the team regarding this immediately.

**Comment by Sh1d0w** [comment]
@jonathansampson Thanks for the quick response.

Just an update from me - it seems just now I've finally got the counter updated and now the count matches what it should be. I was kinda confused as it was more than 24 hours after the ads appeared and the counter was not updated, but all looks good now. I saw there was some downtime on your ads server and I guess that was really the cause.

Thanks for your time and keep up the good work! I am going to close this now.
